tests/integration/tasks/test_ruleset.py

```diff
@@ -23,9 +23,9 @@
 from aap_eda.core.enums import ActivationStatus, RestartPolicy
 from aap_eda.services.ruleset.activate_rulesets import ActivateRulesets
 from aap_eda.tasks.ruleset import (
-    activate,
+    _activate,
```

Comment:
@bzwei Why do we have to import private functions, can we not make them private if you want to import them into other files.

Reply:
function `activate` is to place `_activate` into queue. `_activate` is imported here purely for testing purpose. I guess we could lift it to public if it is a rule or best practice for import.

Reply:
Why we wouldn't test private functions?